docs/contributing/README.md

```diff
@@ -0,0 +1,25 @@
+# Contributing to RVM
+
+You are very warmly welcome to help. Please refer to our [Code of Conduct](code-of-conduct.md) and make sure you follow our [formatting guidelines](formatting.md).
+
+Any and all contributions offered in any form, past present or future are understood to be in complete agreement and acceptance with the [Apache License](LICENSE) v2.0.
+
+## Common workflows
+
+* [Local development environment](local.md)
+* [Adding new Ruby version](new-ruby-version.md)
+* [Signing releases](signing-releases.md)
+* [Testing](testing.md)
+
+## RVM 2.0 work announcement
+
+Work on RVM 2.0 has started, check the plan here:
+https://docs.google.com/document/d/1xW9GeEpLOWPcddDg_hOPvK4oeLxJmU3Q5FiCNT7nTAc/edit?usp=sharing
+
+To be able to do the RVM 2.0 work we need to stop working on RVM 1.x -
+this means no more feature requests will be accepted.
+Development on RVM 1.x will continue to provide bug fixes and
+support for new ruby versions as long as RVM 2.0 will not become stable.
```

Comment:
I think "as long as RVM 2.0 will not become" should be changed to "until RVM 2.0 is"